runtime/jailer_integ_test.go

```diff
@@ -79,3 +80,17 @@ func testJailer(t *testing.T, jailerConfig *proto.JailerConfig) {
 		require.NoError(err, "failed to delete a container")
 	}()
 }
+
+func TestJailerCPUSet_Isolated(t *testing.T) {
+	prepareIntegTest(t, withJailer())
+
+	t.Run("Jailer NumaNode", func(t *testing.T) {
```

Comment:
Can you update the name to reflect the new function name?

Reply:
Done!